### ISSUE TITLE
Proposal of game mode

### ISSUE THREAD

**Issue by akeix** [Vocaluxe/Vocaluxe#363]
Is it possible duo mode even if the song does not predict it.
The song is divided between the two players.

Here's an example: https://store.playstation.com/en-us/product/EP4062-CUSA05101_00-VGKMLETSSING2017

I did not find an example on Youtube.

**Comment by flokuep** [comment]
Yeah, would be possible. Need to think about how _good_ dividing could work. I'm currently not sure, if this could be done easily with current implementation, because duets are 'own' songs.

**Comment by AlecScudder** [comment]
I create my own duet TXT file for every song. It mimics the old "cooperative play" mode of SingStar. I divide the songs so P1 sings alone about 1/3 of the time, P2 sings alone about 1/3 of the time, and then P1 and P2 sing together about 1/3 of the time. 

It is an art getting this right for each song so I think it would be very hard to automate. Song structure are all so different, and to divide evenly may depend on number of lines, syllables, and length of notes. I fear if you try to automate you'd end up with one player getting most of the "good" parts and it wouldn't be fun.  However, if someone thinks it can be done I would be happy to try to help with the logic.

If anyone is interested in the way I create these files let me know. I have an Access database that I use to read the (non-duet) TXT file. Then it has buttons for dividing up the duet parts line by line, with running counts of syllables/lines by player. Finally it has a visual way to set golden notes.
